### ISSUE TITLE
Can not install `0.0.2`

### ISSUE THREAD

**Issue by thearabbit** [zebra-lucky/meteor-aggregate-retyped#1]
I tried install new `0.0.2`, but can not
```js
meteor add zebralucky:aggregate-retyped

Changes to your project's package version selections:

zebralucky:aggregate-retyped  added, version 0.0.1


zebralucky:aggregate-retyped: Mongo.Collection aggRetyped aggregate method
```

**Comment by thearabbit** [comment]
I base on `METEOR@1.8.0.2`

**Comment by zebra-lucky** [comment]
Published version 0.0.3:
> [master 25bef66] version 0.0.3, add README.md, fix version restrictions

Do it fix installation problem?

**Comment by thearabbit** [comment]
👍
